mkdocs.yml

```diff
@@ -1,6 +1,6 @@
 site_name: django-user-management
 pages:
-- [index.md, Home]
+- [README.md, Home]
```

Comment:
seems like it's not working if you run `make run-doc` we have a listing directory instead of a web page. Maybe `mkdocs` has another way to do it?